saleor/plugins/webhook/shipping.py

```diff
@@ -127,18 +128,14 @@ def get_excluded_shipping_methods_from_response(
     excluded_methods = []
     for method_data in response_data.get("excluded_methods", []):
         try:
-            raw_id = method_data["id"]
-            typename, _id = graphene.Node.from_global_id(raw_id)
-            if typename == APP_ID_PREFIX:
-                method_id = raw_id
-            elif typename == "ShippingMethod":
-                method_id = _id
-            else:
+            type_name, method_id = from_global_id_or_error(method_data["id"])
+            if type_name not in (APP_ID_PREFIX, "ShippingMethod"):
                 logger.warning(
-                    "Invalid type received. Expected ShippingMethod, got %s", typename
+                    "Invalid type received. Expected ShippingMethod, got %s", type_name
```

Comment:
Please rewrite it to `f-string`

Reply:
Not sure if this should be rewritten as Sentry is able to process the logs with such format - `fun(msg, arg1,arg2)`. This case is for `warning`, but we should the same format for all logs. Not sure if f-string will be also handled correctly by Sentry (`msg, arg1, arg2` is aggregated on Sentry side by msg, and showed as a single error.)